src/generate/generator.rs

```diff
@@ -90,12 +90,12 @@ impl Generator {
     }
 
     /// Export the current stream to a file, making it very easy to debug the output of a derive macro.
-    /// This will try to find rust's `target` file, and write `target/<name>_<file_postfix>.rs`.
+    /// This will try to find rust's `target` directory, and write `target/generated/<crate_name>/<name>_<file_postfix>.rs`.
     ///
     /// Will return `true` if the file is written, `false` otherwise.
     ///
-    /// The outputted file is unformatted. Use `cargo fmt -- target/<file>.rs` to format the file.
-    pub fn export_to_file(&self, file_postfix: &str) -> bool {
+    /// The outputted file is unformatted. Use `cargo fmt -- target/generated/<crate_name>/<file>.rs` to format the file.
+    pub fn export_to_file(&self, crate_name: &str, file_postfix: &str) -> bool {
```

Comment:
It would be nice if we can automatically derive the `crate_name` ourselves, but I'm not sure if there's a way to do that

Reply:
I gave this some thought and tried a couple of experiments, but couldn't come up with anything. :( Seems like it might not be possible without maybe exporting a macro or something that starts getting hacky.